endorse/endorse.go

```diff
@@ -58,6 +58,9 @@ type Context struct {
 	// Timestamp is what time will be reported in the golden measurement document.
 	Timestamp time.Time
 	VCS       VersionControl
+	// VCSs is an array of VCSs to use to commit the signatures. Needed for transitioning
+	// between systems.
+	VCSs []VersionControl
```

Comment:
Having both a single and an array is pretty confusing. Prefer just having the array, but if this is necessary add a fat comment explaining what's happening.

Reply:
Added.